allennlp/data/tokenizers/sentence_splitter.py

```diff
@@ -44,8 +47,10 @@ def __init__(self,
         self.spacy = get_spacy_model(language, parse=not rule_based, ner=False, pos_tags=False)
         if rule_based:
             # we use `sentencizer`, a built-in spacy module for rule-based sentence boundary detection.
-            if not self.spacy.has_pipe('sentencizer'):
-                sbd = self.spacy.create_pipe('sentencizer')
+            # depending on the spacy version, it could be called 'sentencizer' or 'sbd'
+            sbd_name = 'sbd' if spacy.__version__ < '2.1' else 'sentencizer'
```

Comment:
If you changed this to a unique name which didn't depend on the version, you wouldn't have to disable the tests that only error because of the loading mismatch, rather than the changed model (the constituency parser test and the text_classification reader test).